src/Features/registration.feature

```diff
@@ -23,50 +23,50 @@ Feature: When an user needs to register for a new token
     Then I should see "urn:oasis:names:tc:SAML:2.0:status:Success"
 
     And the logs are:
-      | level  | message                                                                                                                       | sari    |
+      | level  | message                                                                                                                              | sari    |
```

Comment:
Idem here. Please revert the re-formatting :bow:

Reply:
see https://github.com/OpenConext/Stepup-tiqr/pull/143#discussion_r852874098